<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WIN-NFDCINOH1RK" ignored="0" name="Group1" tests="4" failures="2" timestamp="2020-06-17T07:18:37 UTC" time="9.177" errors="0">
  <testcase name="TestAssert" time="0.026" classname="POM.GoogleDemo"/>
  <testcase name="TestAssertB" time="0.004" classname="POM.GoogleDemo"/>
  <testcase name="TestAssertB1" time="0.002" classname="POM.GoogleDemo">
    <failure type="java.lang.AssertionError" message="expected [some wrong Titile] but found [Google]">
      <![CDATA[java.lang.AssertionError: expected [some wrong Titile] but found [Google]
at POM.GoogleDemo.TestAssertB1(GoogleDemo.java:40)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- TestAssertB1 -->
  <testcase name="ValidLogin__fact" time="1.344" classname="POM.VerifyLogin_PgFactory">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at POM.VerifyLogin_PgFactory.ValidLogin__fact(VerifyLogin_PgFactory.java:43)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- ValidLogin__fact -->
</testsuite> <!-- Group1 -->
